js/src/modals/ExecuteContract/executeContract.js

```diff
@@ -334,17 +399,25 @@ class ExecuteContract extends Component {
           });
       })
       .then((txhash) => {
-        this.setState({ sending: false, step: finalstep, txhash, busyState: 'Your transaction has been posted to the network' });
+        this.setState({
+          sending: false,
+          step: finalstep,
+          txhash,
+          busyState:
+            <FormattedMessage
+              id='executeContract.busy.posted'
+              defaultMessage='Your transaction has been posted to the network' />
+        });
       })
       .catch((error) => {
         console.error('postTransaction', error);
         store.dispatch({ type: 'newError', error });
       });
   }
 
-  onGasEditClick = () => {
+  onAdvancedClick = () => {
```

Comment:
[tiny grumble] If this was called `toggleAdvancedOptions`, it would resemble the `advancedOptions` flag which it toggles. Also, `onCheck={ toggleAdvancedOptions }` is more readable.